src/MasterSite.php

```diff
@@ -275,6 +275,41 @@ public function set_featured_image( $post_id, $post, $update ) {
 		}
 	}
 
+	/**
+	 * Generates hreflang metadata from countries.json template.
+	 */
+	public function generate_hreflang_meta() {
+
+		$countries = wp_cache_get( 'countries' );
+
+		if ( false === $countries ) {
+			$request   = wp_remote_get( $this->theme_dir . '/templates/countries.json' );
```

Comment:
I'm confused about this, do we have to do an http request for what looks like a local file ? Does `$json = file_get_contents( get_template_directory() .  '/templates/countries.json' )` works ?

Reply:
Indeed! `file_get_contents` was also my first thought, but `phpcs` was unhappy suggesting `wp_remote_get`. That was because I used `$this->theme_dir`. Using `get_template_directory` instead made `phpcs` happy again.